ProgramParsing/Environment/UpdateDegreeRequirement.py

```diff
@@ -6,35 +6,47 @@
+                    "https://ugradcalendar.uwaterloo.ca/group/ENV-School-Environment-Resources-Sustainability/?ActiveDate=9/1/",
+                    "https://ugradcalendar.uwaterloo.ca/group/ENV-Department-of-Geography-and-Environmental-Mgmt/?ActiveDate=9/1/",
+                    "https://ugradcalendar.uwaterloo.ca/group/ENV-School-of-Planning/?ActiveDate=9/1/"]
 
 #for now we are only parsing the plans
-pages = ["https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Co-operative-Planning",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Env-Res-Sus-Env-Res-Stud-Hons-Reg-Co-op",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Bachelor-of-Knowledge-Integration-1",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Environment-Business-Co-op-and-Reg",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-International-Development",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Geography-Environmental-Management-3-Yr-Gen",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Geography-Environmental-Management-4-Yr-Honour",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Geography-and-Aviation-Regular",
-         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Geomatics-Regular-and-Co-op"]
+pages = ["https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Co-operative-Planning/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Env-Res-Sus-Env-Res-Stud-Hons-Reg-Co-op/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Bachelor-of-Knowledge-Integration-1/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Environment-Business-Co-op-and-Reg/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-International-Development/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Geography-Environmental-Management-3-Yr-Gen/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Geography-Environmental-Management-4-Yr-Honour/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Geography-and-Aviation-Regular/?ActiveDate=9/1/",
+         "https://ugradcalendar.uwaterloo.ca/page/ENV-Honours-Geomatics-Regular-and-Co-op/?ActiveDate=9/1/"]
 
 root = "http://ugradcalendar.uwaterloo.ca/"
 
-def fetch_degree_req(path):
+def fetch_plan(path, year):
     """
          path: str
          Script to download all html pages for degree req
          return:
     """
     link_to_ignore = ["Overview", "Accelerated Master's"]
+    # uses the correct set of plans depending on the year
+    if year >= 2021:
+        plans = plans_2021_newer
+    else:
+        plans = plans_2020_older
 
     # fetch programs
-    http = urllib3.PoolManager()
+    http = urllib3.PoolManager(cert_reqs='CERT_NONE')
     for plan in plans:
-        response = http.request('GET', plan)
+        response = http.request('GET', plan + str(year))
```

Comment:
I would add /?ActiveDate=9/1/ + str(year) here. This is because /? is a param you request to the website, so we want it here with the years. Makes the code more readable